app/cdap/components/WrangleHome/Components/WrangleCard/iconStore/ImportDataIcon.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+
+export const ImportDataIcon = (
```

Comment:
I think here we can maintain all icons in one place(component wise iconStore.tsx)
eg: move them to one file and export it fro there only instead of creating multiple files for multiple icons i.e., AddConnectionIcon,ImportDataIcon